src/main/java/vfyjxf/bettercrashes/BetterCrashesConfig.java

```diff
@@ -44,6 +45,13 @@ public static void syncConfig() {
                 GENERAL,
                 "mclo.gs",
                 "Service to use for uploading crashlogs. Currently, only mclo.gs is currently supported.");
+        issueTrackerURL = config.getString("issueTrackerURL", GENERAL, "", "Link to a bug tracker.");
+        unsupportedMods = Arrays.asList(
+                config.getStringList(
+                        "unsupportedMods",
+                        GENERAL,
+                        new String[] { "Optifine" },
```

Comment:
Sure, I guess, but there there was a [special case for adding Optifine to the list ](https://github.com/GTNewHorizons/BetterCrashes/blob/e31bd1c2093278d9bc3934a3893ecba000fb8a61/src/main/java/vfyjxf/bettercrashes/utils/GuiProblemScreen.java#L255-L257) previously, so that's why I added it to the defaults.

Reply:
Ohh, I just saw your Optifine comment below. I'll look into it.